CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 21.09+ (???)
 ------------------------------------------------------------------------
 - Fix: [#1161] Stuttering vehicle movement due to incorrect vehicle positions.
+- Fix: [#1162] Incorrect signal side placement and cursor hot points.
```

Comment:
Let's make this two fixes, as discussed.

Reply:
Well its really 3 fixes now